docs/compose-file.md

```diff
@@ -744,7 +744,7 @@ then read-write will be used.
 >     - container_name
 >     - container_name:rw
 
-### cpu\_shares, cpu\_quota, cpuset, domainname, hostname, ipc, mac\_address, mem\_limit, memswap\_limit, mem\_swappiness, oom\_score\_adj, privileged, read\_only, restart, shm\_size, stdin\_open, tty, user, working\_dir
+### cpu\_shares, cpu\_quota, cpuset, domainname, dns\_opts, hostname, ipc, mac\_address, mem\_limit, mem\_swappiness, memswap\_limit, oom\_score\_adj, privileged, read\_only, restart, shm\_size, stdin\_open, tty, user, working\_dir
```

Comment:
Shouldn't this be `dns_opt` here and not `dns_opts` to be consistent with the rest of the PR?